site/content/en/docs/Installation/_index.md

```diff
@@ -39,7 +39,7 @@ For all the platforms in Alpha, we would appreciate testing and bug reports on a
 
 ## Agones and Kubernetes Supported Versions
 
-Agones will support 3 releases of Kubernetes, targeting the newest version as being the [default version in the GKE Rapid channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions). However, we will ensure that at least one of the 3 versions chosen for each Agones release is supported by each of the major cloud providers (EKS and AKS). The vendored version of client-go will be aligned with the middle of the three supported Kubernetes versions. When a new version of Agones supports new versions of Kubernetes, it is explicitly called out in the [release notes](https://agones.dev/site/blog/releases/).
+Agones will support 3 releases of Kubernetes, targeting the newest version as being the [latest available version in the GKE Rapid channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions). However, we will ensure that at least one of the 3 versions chosen for each Agones release is supported by each of the major cloud providers (EKS and AKS). The vendored version of client-go will be aligned with the middle of the three supported Kubernetes versions. When a new version of Agones supports new versions of Kubernetes, it is explicitly called out in the [release notes](https://agones.dev/site/blog/releases/).
```

Comment:
The content here feels very similar to the otther, should we consolidate on one place or the other to specify the version support policy? It seems like you link out to this doc above, seems reasonable to de-dupe to the agones site documentation.

Reply:
I feel the other doc (the CONTRIBUTION.md) is more targeting to the contributor and maintainer of the project, while this Installation doc is facing to the user. So I'm hesitating to expose detailed information about the maintenance steps of version upgrade in this doc since user may and should not care about that.